stko/molecular/conversion/ase.py

```diff
@@ -0,0 +1,43 @@
+"""
+Functions for molecule conversion between stk and ASE.
+
+"""
+
+import logging
+from ase import Atoms, Atom
+
+logger = logging.getLogger(__name__)
+
+
+class ASEConverter:
```

Comment:
I don't think it makes sense for this to be a seperate class. What I think we could do is change the from_stk function to a converter function in a Molecule class we implement in our code. Maybe called to_ase_atoms()? What do you think?

Reply:
I actually agree with this statement - we can write a class that contains an stk.Molecule and adds these methods. Will make issue and do this first.